packages/flutter/lib/src/widgets/animated_child_switcher.dart

```diff
@@ -53,17 +148,69 @@ class AnimatedChildSwitcher extends StatefulWidget {
   /// [duration].
   final Widget child;
 
+  /// The duration over which to perform the cross fade using [FadeTransition].
+  final Duration duration;
+
   /// The animation curve to use when fading in the current widget.
```

Comment:
The animation curve to use when transitioning in [child].

Reply:
Done, thanks for catching these.